### PR TITLE
[gui-tests][full-ci] Added test to search for user and group collaborator having same name

### PR DESCRIPTION
**Pull request by HanaGemela**
From manual test case 2319

### CODE REVIEW

File: test/gui/tst_sharing/test.feature [hunk 1]
```diff
@@ -111,6 +111,16 @@ Feature: Sharing
...
+    Scenario: autocomplete offers a list of users followed by a list of groups
+        Given user "admin" has been created on the server with default attributes and without skeleton files
+        And user "Alice" has uploaded file with content "ownCloud test text file" to "textfile.txt" on the server
+        And user "Alice" has set up a client with default settings
+        When the user opens the sharing dialog of "textfile.txt" using the client-UI
+        And the user searches for collaborator with autocomplete characters "Adm" using the client-UI
+        Then the following users should be listed as suggested collaborators:
+            | user  |
+            | Admin |
+            | Admin |
```
Comment (saw-jan) [inline test/gui/tst_sharing/test.feature:123]: We were told that sharing dialog will be dropped. So do we still want to cover the test for it?
See https://github.com/owncloud/client/pull/10959#issuecomment-1602479049
CC @TheOneRing

Comment (HanaGemela) [reply test/gui/tst_sharing/test.feature:123]: That's just an oc10 scenario

File: test/gui/tst_sharing/test.feature [hunk 2]
```diff
@@ -111,6 +111,16 @@ Feature: Sharing
             | TestUser1 |
             | TestUser2 |
 
+    Scenario: autocomplete offers a list of users followed by a list of groups
+        Given user "admin" has been created on the server with default attributes and without skeleton files
```
Comment (saw-jan) [inline test/gui/tst_sharing/test.feature:115]: user `admin` will be there already
```suggestion
```

File: test/gui/tst_sharing/test.feature [hunk 3]
```diff
@@ -111,6 +111,16 @@ Feature: Sharing
...
+        And user "Alice" has uploaded file with content "ownCloud test text file" to "textfile.txt" on the server
```
Comment (saw-jan) [inline test/gui/tst_sharing/test.feature:116]: we have changed the step name
```suggestion
        And user "Alice" has uploaded file with content "ownCloud test text file" to "textfile.txt" in the server
```

File: test/gui/tst_sharing/test.feature (hunk 1 above)
Comment (saw-jan) [inline test/gui/tst_sharing/test.feature:123]:
```suggestion
            | admin         |
            | admin (group) |
```

File: test/gui/tst_sharing/test.feature [hunk 4]
```diff
@@ -111,6 +111,15 @@ Feature: Sharing
...
 
+    Scenario: autocomplete offers a list of users followed by a list of groups
+        And user "Alice" has uploaded file with content "ownCloud test text file" to "textfile.txt" in the server
+        And user "Alice" has set up a client with default settings
+        When the user opens the sharing dialog of "textfile.txt" using the client-UI
+        And the user searches for collaborator with autocomplete characters "Adm" using the client-UI
+        Then the following users should be listed as suggested collaborators:
+            | user  |
+            | admin         |
+            | admin (group) |
```
Comment (saw-jan) [inline test/gui/tst_sharing/test.feature:122]:
```suggestion
            | user          |
            | admin         |
            | admin (group) |
```

**Review by saw-jan — COMMENTED** [review]
others look good to me.

**Comment by saw-jan** [comment]
Could you change PR title to this?
`[gui-tests][full-ci] Added test to search for user and group collaborator having same name`

**Review by saw-jan — APPROVED** [review]

**Pull request merged**